salt/beacons/wtmp.py

```diff
@@ -26,7 +26,7 @@
 
 __virtualname__ = 'wtmp'
 WTMP = '/var/log/wtmp'
-FMT = 'hi32s4s32s256shhiii4i20x'
+FMT = b'hi32s4s32s256shhiii4i20x'
```

Comment:
Same comment for this file as in the btmp beacon (see [here](https://github.com/saltstack/salt/pull/45238/files#diff-6923ee45f755c24a8dbd1ab23dd569efR29)).